docs/userguide/simulation.md

```diff
@@ -1,4 +1,121 @@
 # Simulation
 
-This page will soon contain information about simulations.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+BayBE offers multiple functionalities to "simulate" experimental campaigns with a given lookup mechanism. This user guide briefly introduces how to use the methods available in our [simulation submodule](baybe.simulation).
+
+For a wide variety of applications of this functionality, we refer to the corresponding [examples](../../examples/Backtesting/Backtesting).
+
+## Terminology: What do we mean by "Simulation"?
+
+The term "simulation" can have two slightly different interpretations, depending on the applied context.
+
+1. It can refer to "backtesting" a particular experimental campaign on a fixed finite dataset.
+In this context, "simulation" means investigating what experimental trajectory we would have observed if we had applied the recommender in a certain defined context and restricted the possible parameter configurations to those contained in the dataset.
+
+2. It can refer to the simulation of an *actual* DOE loop, i.e., recommending experiments and retrieving the corresponding measurements, where the loop closure is realized in the form of a callable (black-box) function that can be queried during the optimization to provide target values.
+
+In BayBE, the simulation submodule allows a wide range of use cases and can even be used for "oracle predictions".
+This is enabled by the proper use of the `lookup` functionality, which allows to either use fixed data sets, analytical functions, and general callbacks for retrieving target function values. 
+
+## The lookup functionality
+
+All functions require a `lookup` which is used to close the loop and return target values for points in the search space.
+It can be provided in the form of a dataframe or a `Callable`.
+
+```{note}
+Technically, the `lookup` can also be `None`. This results in the simulation producing fake results which is not useful for real applications and thus not discussed further.
```

Comment:
`not useful` is maybe a bit to negatively worded here, it is for instance useful for simply testing code before you care about implementing the lookup. maybe just remove that subsentence so it says plain `fake results, which is not discussed further.` or beautify it with HIC

also prob replace `fake` by `random`